src/main/js/controls/Graph/GraphConfig.js

```diff
@@ -164,7 +168,42 @@ export const validateContent = (content, input) => {
         throw new Error(errors.THROW_MSG_NON_UNIQUE_PROPERTY);
     }
 };
-
+/**
+ * Checks if panning is enabled or not
+ *
+ * @private
+ * @param {object} config - config object used by the graph.
+ * @returns {boolean} returns true of panning enabled else false.
+ */
+export const isPanningModeEnabled = (config) => {
```

Comment:
Same comment as mentioned in `GanttConfig`. If you want we could even move this function to BaseConfig and have Gantt, Graph and Timeline use from there.

Reply:
As per below comment we are not moving it to baseConfig

Reply:
Sorry for the misunderstanding, I was saying that `isPanningModeEnabled` function can be moved but `settingsDictionary` cannot be.

Reply:
ok sure I will move the isPanningModeEnabled function ti BaseConfig